src/scss/partials/_banner.scss

```
@@ -0,0 +1,53 @@
+.banner {
+    .banner__indicators {
```

tìm hiểu thêm về cách sử dụng dấu & trong scss
```suggestion
    &__indicators {
```